### PR TITLE
[12.0] [MIG] l10n_us_form_1099

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
US Form 1099
============

When companies hire others as contractors, the contractor in question may work for themselves, another company or be the whole other company. This is a huge range, but it can be simplified down to "do I report their payment to the IRS as a 1099?".

This module extends the functionality of a partner and allow you to specify 1099 suppliers.

You will still need to manage payment and IRS reporting separately.

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@max3903 @JayVora-SerpentCS Could you please Review it?

**Review by bodedra — APPROVED** [review]

File: l10n_us_form_1099/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Open Source Integrators <https://opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is_1099 = fields.Boolean('Is a 1099')
```
Comment (JayVora-SerpentCS) [inline l10n_us_form_1099/models/res_partner.py]: change to : 'Is a 1099?'

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by nikul-serpentcs** [comment]
@JayVora-SerpentCS Done :+1:

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by JayVora-SerpentCS** [comment]
Thanks @pedrobaeza @max3903

**Comment by nikul-serpentcs** [comment]
@OCA-git-bot Thanks to reminding 5 days notification :smile: . 

@pedrobaeza Thanks

**Pull request merged**